operator/controllers/loki/certrotation_controller.go

```diff
@@ -55,27 +57,26 @@ func (r *CertRotationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	checkExpiryAfter := expiryRetryAfter(rt.TargetCertRefresh)
-	r.Log.Info("Checking if LokiStack certificates expired", "name", req.String(), "interval", checkExpiryAfter.String())
+	ll.Info("checking if certificates expired", "interval", checkExpiryAfter.String())
```

Comment:
I recently wondered if we should remove a few of these outputs that happen every reconciliation and only print output if there's anything "noteworthy". Don't know if this should also happen in this PR though.